traits/tests/test_ctraits.py

```diff
@@ -150,3 +151,35 @@ def test_no_value_test_flag(self):
 
         with self.assertRaises(AttributeError):
             trait.no_value_test_flag = False
+
+    def test_module_constants_exist(self):
```

Comment:
Having tests for private variables is a bit icky; if we update this PR to include the code that makes use of these constants, then we can remove this test (because a failure to export the constants from ctraits.c will then become a functional failure of the code).